src/Python/qsharp-core/qsharp/clients/iqsharp.py

```diff
@@ -127,6 +127,25 @@ def get_workspace_operations(self) -> List[str]:
     def reload(self) -> None:
         return self._execute(f"%workspace reload", raise_on_stderr=True)
 
+    def get_config(self) -> Dict[str, object]:
+        raw = self._execute(f"%config", raise_on_stderr=True)
```

Comment:
is there a reason the functions in this module don't have docstrings?

Reply:
That was my intent with initial versions, yeah; that separation allows for the mock client to be substituted into the package easily during unit tests and such.

Reply:
Thanks both! I would still argue we could at some point add docstrings here, but happy to discuss :)

Reply:
I don't think there's an argument against, so much as just that they're being internal-only means we didn't get to it yet.